### ISSUE TITLE
AppImageLauncher is not interpreting on Lubuntu 18.04

### ISSUE THREAD

**Issue by lalitmee** [TheAssassin/AppImageLauncher#232]
Hey Guys,

I have this OS
```
➜  ~ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.3 LTS
Release:        18.04
Codename:       bionic
```

I have tried installing [AppImageLauncher](https://github.com/TheAssassin/AppImageLauncher/releases/download/v1.4.0/appimagelauncher_1.4.0-travis754.git20190802.7b45065.bionic_amd64.deb) so many times. But I don't know why it's not working on my system.

When I do `update-binfmts --display`, the output is this:
```
python3.6 (enabled):
     package = python3.6
        type = magic
      offset = 0
       magic = \x33\x0d\x0d\x0a
        mask = 
 interpreter = /usr/bin/python3.6
    detector = 
llvm-6.0-runtime.binfmt (enabled):
     package = llvm-6.0-runtime
        type = magic
      offset = 0
       magic = BC
        mask = 
 interpreter = /usr/bin/lli-6.0
    detector = 
python2.7 (enabled):
     package = python2.7
        type = magic
      offset = 0
       magic = \x03\xf3\x0d\x0a
        mask = 
 interpreter = /usr/bin/python2.7
    detector = 
jar (enabled):
     package = openjdk-11
        type = magic
      offset = 0
       magic = PK\x03\x04
        mask = 
 interpreter = /usr/bin/jexec
    detector = 
```

Please help me. Thank you so much.

**Comment by TheAssassin** [comment]
Please post a log of the installation with `dpkg`.

**Comment by lalitmee** [comment]
Hey @TheAssassin, this is the log while installing with `dpkg`

```
➜  Downloads dpkg appimagelauncher_1.4.0-travis754.git20190802.7b45065.bionic_amd64.deb 
[sudo] password for lalit: 
Selecting previously unselected package appimagelauncher.
(Reading database ... 251823 files and directories currently installed.)
Preparing to unpack appimagelauncher_1.4.0-travis754.git20190802.7b45065.bionic_amd64.deb ...
Unpacking appimagelauncher (1.4.0-travis754~git20190802.7b45065+bionic) ...
Setting up appimagelauncher (1.4.0-travis754~git20190802.7b45065+bionic) ...
Installing AppImageLauncher as interpreter for AppImages
+ update-binfmts --package appimage --install appimage-type1 /usr/bin/AppImageLauncher --magic 'AI\x01' --offset 8
+ update-binfmts --package appimage --install appimage-type2 /usr/bin/AppImageLauncher --magic 'AI\x02' --offset 8
Processing triggers for gnome-menus (3.13.3-11ubuntu1.1) ...
Processing triggers for desktop-file-utils (0.23-1ubuntu3.18.04.2) ...
Processing triggers for mime-support (3.60ubuntu1) ...
Processing triggers for man-db (2.8.3-2ubuntu0.1) ...
Processing triggers for shared-mime-info (1.9-2) ...
```

**Comment by TheAssassin** [comment]
Have you tried calling the two `update-binfmts` calls yourself as well?

```
update-binfmts --package appimage --install appimage-type1 /usr/bin/AppImageLauncher --magic 'AI\x01' --offset 8
update-binfmts --package appimage --install appimage-type2 /usr/bin/AppImageLauncher --magic 'AI\x02' --offset 8
```

Have you tried to reboot? Sometimes systemd's weird binfmt system can get in one's way. For instance, it requires a reboot to clean up stale entries, etc...

**Comment by TheAssassin** [comment]
Please `ls -al /proc/sys/binfmt_misc`.

**Comment by lalitmee** [comment]
Hey @TheAssassin, I did the last command and I am getting no result
```
➜  ~ ls -al /proc/sys/binfmt_misc
ls: cannot access '/proc/sys/binfmt_misc': No such file or directory
```

**Comment by TheAssassin** [comment]
Something's clearly broken on your system.

Please try the reboot I suggested in the previous comment, which you don't seem to have seen.

**Comment by lalitmee** [comment]
@TheAssassin, you mean I have to call those two commands manually? :thinking:

**Comment by TheAssassin** [comment]
No, as we just checked `binfmt_misc` doesn't seem to work at all on your system. You can double-check with `lsmod | grep binfmt`. You could try to reboot; after all you have the required packages installed. If that also doesn't work, please refer to an Ubuntu forum or something like that.

TL;DR: This bug is not in AppImageLauncher, it's your system lacking support for `binfmt_misc` at all.

**Comment by lalitmee** [comment]
```
➜  ~ lsmod | grep binfmt
binfmt_misc            20480  1
```
I am getting this.

**Comment by TheAssassin** [comment]
Seems like the module is loaded at least.

Look, I can't provide any regular Ubuntu support. Please try to figure out the solution yourself. Sorry.

**Comment by lalitmee** [comment]
@TheAssassin, no problem. 

Thank you for your help.
